lib/github_service/commands/cross_repo_test.rb

```diff
@@ -207,6 +220,37 @@ def valid?
         true
       end
 
+      def validate_repo_names
+        invalid_repo_names = (@test_repos + @repos).select { |r| r.include?(" ") }.sort.uniq
```

Comment:
For now I just check for a space, because that's the easiest way to handle certain invalid things like forgetting to use commas to separate repo, or using the word `includes` instead of `including`. Eventually we can fully check all valid characters if we wanted.